node-require/calculate.js

```diff
@@ -0,0 +1,19 @@
+const add = require('./add');
+const subtract = require('./subtract');
+const divide = require('./divide');
+const multiply = require('./multiply');
+
+switch (process.argv[3]) {
```

Comment:
It would make the rest of your code simpler if you would pull out the arguments into variables and do the conversion to number here. Then you wouldn't need to use `Number(x)` everywhere.